src/NzbDrone.Core/ImportLists/HttpImportListBase.cs

```diff
@@ -239,7 +239,7 @@ protected virtual ValidationFailure TestConnection()
             {
                 _logger.Warn(ex, "Unable to connect to import list");
 
-                return new ValidationFailure(string.Empty, "Unable to connect to import list, check the log for more details");
+                return new ValidationFailure(string.Empty, "Unable to connect to import list, check the log above the ValidationFailure for more details");
```

Comment:
Maybe something like `check the log surrounding this error for details`, that along with the actual error message will give them a better starting point.


```suggestion
                return new ValidationFailure(string.Empty, $"Unable to connect to import list: {ex.Message}. Check the log surrounding this error for details");
```